docs/manual/install/ubuntu.rst

```diff
@@ -2,7 +2,9 @@
 Installing on Debian or Ubuntu
 ==============================
 
-On recent Ubuntu (17.04 or greater) and Debian unstable versions, there are
+Note: As of Ubuntu 20.04 (Focal Fossa), the package has been outdated and removed from the Ubuntu's official package list. Users are advised to follow the instructions of :ref:`installing-from-source`.
```

Comment:
Can you line wrap this to 80 chars? Otherwise lgtm, thanks!

Reply:
Done!